readme.md

```diff
@@ -175,6 +175,38 @@ Default: `process.cwd()`
 
 [Inform iTerm2](https://www.iterm2.com/documentation-escape-codes.html) of the current directory to help semantic history and enable [Cmd-clicking relative paths](https://coderwall.com/p/b7e82q/quickly-open-files-in-iterm-with-cmd-click).
 
+### iTerm.annotation(message, options?)
+
+#### message
+
+Type: `string`<br>
+
+The annotation contents.
+
+#### options
+
+Type: `object`
+
+##### length
+
+Type: `number`
+
+If present, annotate `length` number of cells. Defaults to the remainder of the line.
+
+##### x
+##### y
+
+Type: `number`
+
+Coordinates at which to begin annotation. If specified, `x`, `y`, *and* `length` are all required.
+
+##### hidden
```

Comment:
`hidden` => `isHidden`

Reply:
will hold off on this until [this](https://github.com/sindresorhus/ansi-escapes/pull/17#discussion_r320875447) is clear to me